.gitignore

```diff
@@ -5,3 +5,5 @@ _site/
 
 *.swp
 .idea/
+
+.mypy_cache/
```

Comment:
Where is python used?

Reply:
I opened the logo and organizers grid scripts and got the mypy cache so I added it to gitignore